test/Feature/EmbedDisambiguationSpec.hs

```diff
@@ -194,8 +194,8 @@ spec =
             { matchHeaders = [matchContentTypeJson] }
 
         it "can request two parents with fks" $
-          get "/articleStars?select=createdAt,article(owner),user(name)&limit=1" `shouldRespondWith`
-            [json|[{"createdAt":"2015-12-08T04:22:57.472738","article":{"owner": "postgrest_test_authenticator"},"user":{"name": "Angela Martin"}}]|]
```

Comment:
So what happened here, is that before the rollback changes, it seems like this row was updated and the owner was set permanently to `postgrest_test_authenticator`. Locally, this is still the case, but in CI the row will have been created by the `circleci` sql user. That means we can't test for the owner here, because it might be different in different environments. For this reason I changed it to use the `id` instead.